frida/src/lib.rs

```diff
@@ -19,6 +28,17 @@ impl Frida {
         frida_sys::frida_init();
         Frida {}
     }
+
+    /// Obtains a new instance of device manager
+    pub fn obtain_device_manager(&self) -> DeviceManager {
```

Comment:
Instead of using `obtain_device_manager`, we use an `obtain` function on the appropriate class that takes a reference to `Frida`. See `Interceptor` for an example: https://docs.rs/frida-gum/0.6.3/frida_gum/interceptor/struct.Interceptor.html#method.obtain